app/models/user.rb

```diff
@@ -284,7 +284,7 @@ def inactive_message
     I18n.t(:inactive_html, path_to_contact: '/contact').html_safe
   end
 
-  def avatar_url(size=nil)
+  def avatar_url(size=nil, avatar_kind)
```

Comment:
Cool, so you should be able to fix your error by making this `avatar_kind=nil`

Reason being we call this method elsewhere with no arguments, so the method needs to function without having anything passed in.

Other than that this fix looks like it should work. :tropical_fish: